client/components/Reviews/styles/ReviewsComponent.styled.js

```diff
@@ -4,4 +4,8 @@ export const Container = styled.div`
   display: grid;
   grid-template-columns: 30% 70%;
   margin: 0 50px;
+  height: 100vh;
```

Comment:
Consider a default export instead of an in-line export for style-files that contain only one exported styled-component.